src/move_basic.cpp

```diff
@@ -199,43 +172,43 @@ MoveBasic::MoveBasic(): tfBuffer(ros::Duration(3.0)),
 {
     ros::NodeHandle nh("~");
 
-    nh.param<double>("min_angular_velocity", minAngularVelocity, 0.05);
```

Comment:
Now that the names and meaning of params will be changed, its important we also update the documentation on README and ROS wiki

Reply:
Yes yes of course, I will definitely do that before the merge - actually at the same time hah